vta/python/vta/__init__.py

```diff
@@ -29,8 +30,18 @@
 
 __version__ = "0.1.0"
 
-# do not from tvm import topi when running vta.exec.rpc_server
-# to maintain minimum dependency on the board
-if sys.argv[0] not in ("-c", "-m"):
+_LOG = logging.getLogger(__name__)
+
+# do not from tvm import topi when __main__ is in vta_onboard
+# to maintain minimum Python dependencies on the board.
+# TODO(vta-team): move board-only logic imported above into vta_onboard.
```

Comment:
This is likely no longer needed after my update https://github.com/apache/tvm/blob/main/vta/python/vta/__init__.py#L36

Reply:
ah oops this was a bad merge, fixing